rl_loop.py

```diff
@@ -29,10 +29,10 @@
 import shipname
 
 # How many games before the selfplay workers will stop trying to play more.
-MAX_GAMES_PER_GENERATION = 10000
+MAX_GAMES_PER_GENERATION = 20000
```

Comment:
we have several models with more than this so we may have self-play pauses if you choose this number

Reply:
Ack.  We're not using py for selfplay anymore so /shruggie